src/app/components/client/ExposureCard.module.scss

```diff
@@ -5,9 +5,9 @@
   --exposureDetailsIconWidth: 25px;
 
   font: $text-body-sm;
+  background-color: $color-white;
   border: 2px solid rgba($color-purple-70, 0.2);
   border-radius: $border-radius-md;
-  max-width: $content-xl;
```

Comment:
question (non-blocking): Why this change?

Reply:
Otherwise there was empty space to the right of the cards, which I didn't see in the designs (nor anything to indicate that they should have a max width). So

![image](https://github.com/mozilla/blurts-server/assets/4251/53fd3ee1-e8ae-4cfa-8a03-0e57beef7461)

vs

![image](https://github.com/mozilla/blurts-server/assets/4251/ba9f819c-4f03-4945-9e03-4bd6e647422d)